refinery/data_set_manager/tests.py

```diff
@@ -2000,36 +2006,71 @@ def test_prepare_node_remote_datafile_source(self):
         self.file_store_item.datafile.delete()
         self.file_store_item.source = "http://www.example.com/test.txt"
         self.file_store_item.save()
-
         self._prepare_index(
             self.node,
-            expected_download_url=self.file_store_item.get_datafile_url()
+            expected_download_url=self.file_store_item.source
         )
 
     def test_prepare_node_missing_datafile(self):
         self.file_store_item.datafile.delete()
         self.file_store_item.save()
         with mock.patch.object(
-                FileStoreItem,
-                "get_import_status",
-                return_value=SUCCESS
+            FileStoreItem,
+            "get_import_status",
+            return_value=SUCCESS
         ) as get_import_status_mock:
-            self._prepare_index(self.node, expected_download_url="N/A")
+            self._prepare_index(self.node, expected_download_url=NOT_AVAILABLE)
             self.assertTrue(get_import_status_mock.called)
 
-    def test_prepare_node_pending_file_import_task(self):
+    def test_prepare_node_pending_yet_existing_file_import_task(self):
+        self.file_store_item.datafile.delete()
+        self.file_store_item.save()
+        with mock.patch.object(
+            FileStoreItem,
+            "get_import_status",
+            return_value=PENDING
+        ):
+            self._prepare_index(self.node, expected_download_url=PENDING)
```

Comment:
Do you want to make an assertion in this test? Or maybe that's in `_prepare_index`?